chunky/src/java/se/llbit/chunky/model/BlockModel.java

```diff
@@ -3,9 +3,19 @@
 import se.llbit.chunky.plugin.PluginApi;
 import se.llbit.chunky.renderer.scene.Scene;
 import se.llbit.math.Ray;
+import se.llbit.math.Vector3;
+
+import java.util.List;
+import java.util.Random;
 
 @PluginApi
 public interface BlockModel {
 
   boolean intersect(Ray ray, Scene scene);
+
+  int numFaces();
```

Comment:
```suggestion
  int faceCount();
```
I've seen more "*Count()" methods in Java projects (might be easier to read because the thing we count is named first?).